core-storage/src/main/java/com/thoughtworks/ark/core/storage/StorageInterface.kt

```diff
@@ -0,0 +1,31 @@
+package com.thoughtworks.ark.core.storage
+
+import android.graphics.Bitmap
+import java.io.File
+
+interface StorageInterface {
+
+    var path: String?
```

Comment:
once path specified to "Documents"
Can you createFile/loadFile in path "Documents/res/data.txt"? 
With code `createFile("res/data/txt")`, or `fun loadFile("res/data.txt")`

Reply:
Now, we can't do it🤔. Do we need such an interface like that?